automx2/util.py

```diff
@@ -54,6 +54,9 @@ def expand_placeholders(string: str, local_part: str, domain_part: str) -> str:
         PLACEHOLDER_DOMAIN: domain_part,
         PLACEHOLDER_LOCALPART: local_part,
     }
-    for k, v in placeholder_map.items():
-        string = string.replace(k, v)
+    try:
```

Comment:
Why is this try..except needed if it was not needed for other generators?

Reply:
When iterating over XML sometimes the blocks are empty and `element.text` returns NoneType.
Therefore function `string.replace(k, v)` fails since NoneType object has no attribute `replace`

Reply:
Maybe it would not be NoneType if you would sanitize `account` SubElement?

And if not, then catch only NoneType exception